content/setup/local-with-vagrant/_index.md

```diff
@@ -0,0 +1,264 @@
++++
+title = "Local with Vagrant"
+date = 2020-06-08T16:16:15+05:30
+draft = false
+weight = 20
+toc = true
++++
+
+An easy way to try out Tinkerbell locally is via [Vagrant](https://www.vagrantup.com).
+
+Vagrant wraps and glues together different technologies to help you write as
+code repetitive environment mixing virtualization or cloud providers such as
+VirtualBox, libvirt, AWS, GCP with provisioning techniques like Puppet, Ansible,
+Bash.
+
+In this context we use VirtualBox or libvirt as a virtualization platform and
+bash for provisioning all the required tools and configurations.
+
+Following this tutorial, you will end up with a provisioner up and running and a
+worker ready to handle actions and workflows.
+
+## Prerequisite
+
+- The host's processor should support virtualization
```

Comment:
Tell them how to find out or link to where to learn.

Reply:
Do you have some good material on how to do this? `grep -o 'vmx\|svm' /proc/cpuinfo` works for Linux, but I'm not sure about macOS.

Reply:
I would suggest Googling it and linking off to that.

Reply:
(because that's all that is going to happen when a user sees this vague note)